test/test_operations.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2009-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test the operations module."""
+
+from test import UnitTest
+from pymongo import ASCENDING, DESCENDING
+from pymongo.operations import IndexModel
+from pymongo.collation import Collation
+
+
+class TestOperations(UnitTest):
+    """Test Operations module including methods:
+    _Op, _DeleteOp, _UpdateOp,
+    DeleteOne, DeleteMany,
+    InsertOne,
+    ReplaceOne,
+    UpdateOne, UpdateMany,
+    IndexModel,
+    SearchIndexModel"""
+
+    def test_index_model_repr(self):
+        # Based on examples in test_collection.py
+        self.assertRepr(IndexModel("hello"))
+        self.assertRepr(IndexModel([("hello", DESCENDING), ("world", ASCENDING)]))
+        self.assertRepr(
+            IndexModel([("hello", DESCENDING), ("world", ASCENDING)], name="hello_world")
+        )
+
```

Comment:
I meant the comments could replace the blank lines, just to keep things more compact.